usecases/configuration.go

```diff
@@ -116,24 +120,21 @@ func Configure(sys *components.System) ([]json.RawMessage, error) {
 
 	var rawBytes []json.RawMessage // the mbaigo library does not know about the unit asset's structure (defined in the file thing.go and not part of the library)
```

Comment:
I just realised this var is completely useless, after having written the other comments.
Remove this line, and all other places with `rawBytes` should be replaced with `nil`.